examples/langchain.ipynb

```diff
@@ -78,7 +78,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from greptimeai.langchain.callback import GreptimeCallbackHandler\n",
+    "from greptimeai.langchain_callback import GreptimeCallbackHandler\n",
```

Comment:
This is a breaking change and you might want to bump minor version. And we would like to be sure that this breaking change is necessary.